app-script-payslip-sender/Payslip.js

```diff
@@ -0,0 +1,21 @@
+function PayslipSender() {
+  var spreadsheet = SpreadsheetApp.getActive().getSheetByName('Sheet1');
+  var dataRange = spreadsheet.getRange("A2:D2").getValues();
```

Comment:
this will pick just up to 2 entries, right? Need to handle the case where the sheet has hundreds of rows.

Reply:
Column D2 has been changed to D, and it now accepts all values in the D column.